### ISSUE TITLE
Lotte to meet with Henrik Mosegaard to discuss the relationship with stockassessment.org and clarify the different uses.

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-taf/doc#26]

**Comment by neil-ices-dk** [comment]
Arni/Colin to make small overview and especially on reln to stockass.org

**Comment by neil-ices-dk** [comment]
(LOTTE WROTE)
I had a good, constructive meeting with Henrik Mosegaard today concerning TAF and SAO. I first outlined that the ICES approach to the collaboration/co-existence of TAF and SAO was to avoid hindering their individual developments and to keep a transparent and informed dialogue on the developments of the two systems. Henrik was obviously agreeing and stated that the only concerns he had was, that TAF ‘just copied’ a version of the existing SAO, embedding this in TAF and then missing the developments on the SAO. The linkage between the TAF and SAO seem to be somewhat unclear to Henrik, although he could see the highly useful link between all the ICES Data products and TAF.

I explained the anticipated coexistence with TAF as the official repository of ICES assessments and SAO acting as an environment to develop and run assessments within. I tried to ‘cartoon’ it for him, but this can be done much more elegantly I am sure, so we agreed that he and I would outline a meeting between ‘all of us’ to create the ‘arena’ for the coexistence of TAF and SAO. I will have a brief chat with Neil, Colin and Arni on our next TAF update meeting on how to set this up.

So, in conclusion; positive statements regarding the development and co-existence of TAF and SAO and good prospects for a ‘free’ dialogue between our developers and the ones on DTU Aqua.

Cheers, Lotte
